### PR TITLE
Update Docker files

### PR DESCRIPTION
**Pull request by LaRiffle**
Docker files are not uptodate since the last refactor, I'm trying to fix this here.

- [ ] Remove env files from the image building 
- [ ] Switch from dotenv to providing envfile in docker-compose (?)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ version: '3'
 
 services:
   pagai:
-    build:
-      context: ./pagai
+    build: .
```
Comment (simonvadee) [inline docker-compose.yml]: don't you want to add the CONFIG_PATH env variable in the docker-compose file ?

Comment (LaRiffle) [reply docker-compose.yml]: I was thinking of doing like in https://github.com/arkhn/cleaning-scripts/pull/6
Namely  add a `env_file` in the docker-compose, wouldn't this be a better practice?

Comment (simonvadee) [reply docker-compose.yml]: depends on how you intend to parse the config, if you just want to get env variables, then yes using an `env_file` is a better option. However if you want a more structured config (json,yaml,toml...) then you need to parse the config from within your code and provide the `CONFIG_PATH` as an env variable or as argument to your program

Comment (LaRiffle) [reply docker-compose.yml]: Let's keep it a simple env file for the moment, except if you think we should think bigger

**Comment by Jasopaum** [comment]
Stale

**Pull request closed without merging**